site/docs/guides/streaming.mdx

```diff
@@ -232,5 +172,5 @@ You can also create your custom streamer for more sophisticated processing:
 </LanguageTabs>
```

Comment:
`CustomStreamer custom_streamer;` should be `std::shared_ptr<CustomStreamer>`: https://github.com/openvinotoolkit/openvino.genai/blob/fb3f9ca3cdc7bd106f39fc0ba2cefafe37018e3c/src/cpp/include/openvino/genai/llm_pipeline.hpp#L286C61-L286C77

Reply:
Fixed